app/controllers/api/v2/app_groups_controller.rb

```diff
@@ -59,6 +59,78 @@ def cluster_templates
     render json: cluster_templates
   end
 
+  def profile_app
+    profiles = []
+    AppGroup.active.all.each do |appGroup|
+      environment = appGroup.environment
+      helm_infra = appGroup.helm_infrastructure
+
+      infra_values = helm_infra.values.to_json
+      object = JSON.parse(infra_values, object_class: OpenStruct)
+
+      replication_factor = environment == "production" ? 2 : 1
+
+      barito_apps =[]
+      appGroup.barito_apps.where(status:"ACTIVE").each do |barito_app|
+        days = barito_app.log_retention_days
+        if days == nil
+          days = environment == "production" ? 14 : 7
+        end
+        barito_apps << {
+          app_name: barito_app.name,
+          app_log_retention: days,
+        }
+      end
+
+      profiles << {
+        app_group_name: appGroup.name,
+        app_group_cluster_name: helm_infra.cluster_name,
+        app_group_replication_factor: replication_factor,
+        app_group_barito_apps: barito_apps,
+      }
+    end
+    render json: profiles
+  end
+
+  def update_cost
```

Comment:
can we have example the payload for this one?

Reply:
for `update_cost` mas? it should be something like:
```
[
  {
    "id": "309ED3AC-63B3-4780-8C2E-08350B2DCA6B",
    "created_at": "2023-11-10 23:00:00 +0000 UTC m=+0.000000001",
    "app_group_name":  "app group name",
    "app_group_cluster_name":  "rije",
    "app_group_replication_factor":  2,
    "app_name":  "app-name"
    "app_log_retention": 14,
    "app_log_bytes": 889901838120321,
    "calculation_price": 889901838,
    "calculation_log_retention_standard": 14,
    "calculation_replication_factor_standard": 2,
    "calculation_log_ingestion_mode": "BOTH"
  },
]
```

Although the required values are only `app_group_name`, `app_name`, `app_log_bytes`, and `calculation_price`

Reply:
please add somewhere on our documentation about barito API. last time we have this on this [wiki](https://gopay-systems.pages.golabs.io/wiki/products/barito/dev_ops/api_reference.html)
 
since we move to jira and we haven't update it yet. I think it would be better if we have similar one on somewhere [here](https://go-jek.atlassian.net/wiki/spaces/GOCLOUD/pages/1743140113/Logging+for+Dev)

Reply:
ok mas, noted